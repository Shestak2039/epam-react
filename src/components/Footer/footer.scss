@import "../../styles/colors";

.footer {
  height: 70px;
  background-color: $footer-background;
}
