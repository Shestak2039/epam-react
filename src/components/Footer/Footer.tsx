import React from 'react';

import './footer.scss';

const Footer: React.FunctionComponent = () => (
    <footer className="footer" />
);

export default Footer;
