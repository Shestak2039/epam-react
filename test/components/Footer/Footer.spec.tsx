import React from 'react';
import { render } from '@testing-library/react';

import Footer from '../../../src/components/Footer/Footer';

describe('Footer', () => {
    test('renders Footer snapshot', () => {
        const { asFragment } = render(<Footer />);

        expect(asFragment()).toMatchSnapshot();
    });
});
