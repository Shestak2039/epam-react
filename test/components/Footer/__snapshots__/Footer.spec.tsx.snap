// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  />
</DocumentFragment>
`;
